### PR TITLE
Rafael branch

### PR DESCRIPTION
**Pull request by rafaelgtraverso**
new email validation

### CODE REVIEW

File: src/screens/SignInScreen.js [hunk 1]
```diff
@@ -75,6 +75,15 @@ const mapStateToProps = (state) => {
 const mapDispatchToProps = (dispatch) => {
   return{
     sign_in: async ({ email, password }) => {
+      const emailPattern = /^\w+(\.\w+)*@[a-zA-Z_]+?\.[a-zA-Z]{2,3}(\.[a-zA-Z]{2}){0,1}$/;
```
Comment (jgtoriginal) [inline src/screens/SignInScreen.js:78]: For email validation regex I'd go for something as simple as `/\S+@\S+\.\S+/` or just write the email validation method on a function of its own and then instantiate it here.

File: package.json [hunk 2]
```diff
@@ -33,7 +33,10 @@
     "react-native-responsive-screen": "^1.4.1",
     "react-native-safe-area-context": "^3.1.4",
     "react-native-screens": "^2.10.1",
+<<<<<<< HEAD
```
Comment (jgtoriginal) [inline package.json:36]: need to resolve this git conflict.
keep line 37, remove 36-38-39

File: src/screens/SignInScreen.js (hunk 1 above)
Comment (rafaelgtraverso) [reply src/screens/SignInScreen.js]: done

File: src/screens/PasswordRecoveryScreen.js [hunk 3]
```diff
@@ -66,12 +66,17 @@ const mapStateToProps = (state) => {
 const mapDispatchToProps = (dispatch) => {
   return{
     reset_password: async (email) => {
-      try {
-        await auth().sendPasswordResetEmail(email).then(navigate('Signin'));
-      } catch (err) {
-        dispatch(authError('Something went wrong! Please make sure you have enter the same email you use when you register. If you are sure is the correct email, please contact support@cloudingsystems.co.uk'));
-        console.log(err)
+      const emailPattern = /^\w+(\.\w+)*@[a-zA-Z_]+?\.[a-zA-Z]{2,3}(\.[a-zA-Z]{2}){0,1}$/;
```
Comment (jgtoriginal) [inline src/screens/PasswordRecoveryScreen.js:69]: don't forget to put the email regex out of both forms into a file/ method of it's own and instantiate it from there.

Comment (rafaelgtraverso) [reply src/screens/PasswordRecoveryScreen.js]: done!

**Review by jgtoriginal — APPROVED** [review]

**Pull request merged**